tests/libs/datasets/data_source_test.py

```diff
@@ -98,3 +98,30 @@ class NYTimesForTest(nytimes_dataset.NYTimesDataset):
         static=region_static,
     )
     test_helpers.assert_dataset_like(dataset_expected, dataset_read)
+
+
+def test_can_scraper_class_single_provider():
+    cls: data_source.CanScraperBase
```

Comment:
I haven't seen this sort of syntax before, is this the recommended way to show typing for this case?

Reply:
Yes. https://www.python.org/dev/peps/pep-0526/#where-annotations-aren-t-allowed